package.json

```diff
@@ -1,5 +1,6 @@
 {
-  "name": "bookshelf-fsm",
+  "name": "@waypointhomes/bookshelf-fsm",
+  "private": true,
```

Comment:
This is a package. We probably don't need it anymore, but it shouldn't be private.